tests/non-js-resources/subtests/css/subtests/source/parcel/index.md

```diff
@@ -1,12 +1,12 @@
 ---
-result: partial
+result: pass
 issue:
   - url: https://github.com/parcel-bundler/parcel/issues/4320
     fixedSince: 2.0.0-beta.1
     status: closed
   - url: https://github.com/parcel-bundler/parcel/issues/4855
+    fixedSince: 2.0.0-beta.1
```

Comment:
Did you men beta2 here or was it actually fixed in beta1 and we didn’t test it right?

Reply:
Oops, you're right